addons/website_project/controllers/main.py

```diff
@@ -18,21 +18,24 @@ def insert_record(self, request, model, values, custom, meta=None):
             values.setdefault('user_ids', False)
 
         res = super().insert_record(request, model, values, custom, meta=meta)
-        if not (custom or meta) or model.model != 'project.task':
+        if model.model != 'project.task':
             return res
         task = request.env['project.task'].sudo().browse(res)
-        custom = "<b>" + custom.replace('email_from', 'Email').replace(' :', ':</b>').replace('\n', '\n<b>').replace('\r\n<b>', '\r\n')
-        custom_label = "<h4>%s</h4>\n\n" % _("Other Information")  # Title for custom fields
+        custom = custom.replace('email_from', 'Email')
+        custom_label = nl2br_enclose(_("Other Information"), 'h4') # Title for custom fields
         default_field = model.website_form_default_field_id
         default_field_data = values.get(default_field.name, '')
-        default_field_content = "<h4>%s</h4>\n<p>%s</p>" % (default_field.name.capitalize(), html2plaintext(default_field_data))
-        custom_content = (f"{default_field_content} \n\n\n" if default_field_data else '') \
-                        + (f"{custom_label} {custom} \n\n " if custom else '') \
+        default_field_content = nl2br_enclose(default_field.name.capitalize(), 'h4') + nl2br_enclose(html2plaintext(default_field_data), 'p')
+        custom_content = (default_field_content if default_field_data else '') \
+                        + (custom_label + custom if custom else '') \
                         + (self._meta_label + meta if meta else '')
 
         if default_field.name:
+            if default_field.ttype == 'html':
+                custom_content = nl2br(custom_content)
+            task[default_field.name] = custom_content
             task._message_log(
-                body=nl2br_enclose(custom_content, 'p'),
+                body=nl2br(custom_content),
```

Comment:
Wait, with this, `nl2br(custom_content)` is called twice. Is that intended?

Reply:
Rectified the code. I don't think it is required.
I guess I just missed that. Thanks for mentioning it. :slightly_smiling_face: